docs/workflow.md

```diff
@@ -0,0 +1,138 @@
+
+# Workflows
+
+This document describes two different yet common workflows in BKPR using `kubeprod` and `kubecfg`:
```

Comment:
Could we reformat this slightly to be more readable?

`This document describes...kubecfg. These workflows are intended for users of BKPR, most likely developers, DevOps, Kubernetes operators, etc.

- The [basic workflow](link) covers
- The [advanced workflow](link) covers
`